### PR TITLE
✨ Add Hail of Arrows Spell Scroll

### PR DESCRIPTION
**Pull request by shrinerh**

### CODE REVIEW

File: texturepack/assets/foxcraft_dungeon_loot/models/item/spell_scroll_hail_of_arrows.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "parent": "minecraft:item/generated",
+  "textures": {
+    "layer0": "foxcraft_dungeon_loot:item/spell_scroll"
+  }
+}
```
Comment (satyrnidae) [inline texturepack/assets/foxcraft_dungeon_loot/models/item/spell_scroll_hail_of_arrows.json]: end of file EOL

File: data/foxcraft_dungeon_loot/tags/functions/on_tick.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "foxcraft_dungeon_loot:items/hammer_of_sol/on_tick",
     "foxcraft_dungeon_loot:items/totem_of_mysteria/on_tick",
     "foxcraft_dungeon_loot:items/ima_leaf/on_tick",
-    "foxcraft_dungeon_loot:items/totem_of_deilona/on_tick"
+    "foxcraft_dungeon_loot:items/totem_of_deilona/on_tick",
+    "foxcraft_dungeon_loot:items/spell_scroll_hail_of_arrows/on_tick"
   ]
```
Comment (satyrnidae) [inline data/foxcraft_dungeon_loot/tags/functions/on_tick.json:46]: this will definitely need merged after #48

**Review by satyrnidae — CHANGES_REQUESTED** [review]
missing an EOL at the end of the model file, will require a merge w/ main

File: texturepack/assets/foxcraft_dungeon_loot/models/item/spell_scroll_hail_of_arrows.json (hunk 1 above)
Comment (shrinerh) [reply texturepack/assets/foxcraft_dungeon_loot/models/item/spell_scroll_hail_of_arrows.json]: done

File: data/foxcraft_dungeon_loot/tags/functions/on_tick.json (hunk 2 above)
Comment (shrinerh) [reply data/foxcraft_dungeon_loot/tags/functions/on_tick.json:46]: done

**Review by satyrnidae — APPROVED** [review]
looks great :)

**Pull request merged**